src/test/java/org/gridsuite/study/server/NetworkModificationTest.java

```diff
@@ -1532,6 +1521,45 @@ public void testLineAttachToVoltageLevel() {
+        UUID studyNameUserIdUuid = studyEntity.getId();
+        UUID rootNodeUuid = getRootNode(studyNameUserIdUuid).getId();
+        NetworkModificationNode modificationNode = createNetworkModificationNode(studyNameUserIdUuid, rootNodeUuid, VARIANT_ID, "node");
+        UUID modificationNodeUuid = modificationNode.getId();
+
+        String createLinesAttachToSplitLinesAttributes = "{\"lineToAttachTo1Id\":\"line1\",\"lineToAttachTo2Id\":\"line2\",\"attachedLineId\":\"line3\",\"voltageLevelId\":\"vl1\",\"bbsBusId\":\"v1bbs\",\"replacingLine1Id\":\"replacingLine1Id\",\"replacingLine1Name\":\"replacingLine1Name\",\"replacingLine2Id\":\"replacingLine2Id\",\"replacingLine2Name\":\"replacingLine2Name\"}";
+
+        mockMvc.perform(post("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/lines-attach-to-split-lines",
+                        studyNameUserIdUuid, modificationNodeUuid)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(createLinesAttachToSplitLinesAttributes))
+                .andExpect(status().isOk());
+        checkEquipmentCreatingMessagesReceived(studyNameUserIdUuid, modificationNodeUuid);
+        checkEquipmentCreationMessagesReceived(studyNameUserIdUuid, modificationNodeUuid, Collections.EMPTY_SET);
+        checkEquipmentUpdatingFinishedMessagesReceived(studyNameUserIdUuid, modificationNodeUuid);
+
+        mockMvc.perform(put("/v1/studies/{studyUuid}/nodes/{nodeUuid}/network-modification/modifications/{modificationUuid}/lines-attach-to-split-lines",
+                        studyNameUserIdUuid, modificationNodeUuid, MODIFICATION_UUID)
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .content(createLinesAttachToSplitLinesAttributes))
+                .andExpect(status().isOk());
+        checkEquipmentUpdatingMessagesReceived(studyNameUserIdUuid, modificationNodeUuid);
+        checkUpdateEquipmentModificationMessagesReceived(studyNameUserIdUuid, modificationNodeUuid);
+        checkEquipmentUpdatingFinishedMessagesReceived(studyNameUserIdUuid, modificationNodeUuid);
+
+        var requests = TestUtils.getRequestsWithBodyDone(2, server);
+        assertEquals(2, requests.size());
+        Optional<RequestWithBody> creationRequest = requests.stream().filter(r -> r.getPath().matches("/v1/networks/" + NETWORK_UUID_STRING + "/lines-attach-to-split-lines\\?group=.*")).findFirst();
+        Optional<RequestWithBody> updateRequest = requests.stream().filter(r -> r.getPath().matches("/v1/modifications/" + MODIFICATION_UUID + "/lines-attach-to-split-lines-creation")).findFirst();
+        assertTrue(creationRequest.isPresent());
+        assertTrue(updateRequest.isPresent());
+        assertEquals(createLinesAttachToSplitLinesAttributes, creationRequest.get().getBody());
+        assertEquals(createLinesAttachToSplitLinesAttributes, updateRequest.get().getBody());
+    }
```

Comment:
If you want to get a higher coverage "mark" in ModificationType.java, you should test the bogus way, as it was done for LineSplitWithVoltageLevel

Reply:
done